helpers/exceptions.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 module.exports = {
+	mainnetGenesisPublicKey: 'd121d3abf5425fdc0f161d9ddb32f89b7750b4bdb0bff7d18b191d4b4bafa6d4',
```

Comment:
I mean something like that:
```
mainnet: {
    genesisPublicKey: 'd121d3abf5425fdc0f161d9ddb32f89b7750b4bdb0bff7d18b191d4b4bafa6d4'
},
test: {
    genesisPublicKey: 'some_fake_pk...'
}
```
or 
```
genesisPublicKey: {
    mainnet: 'd121d3abf5425fdc0f161d9ddb32f89b7750b4bdb0bff7d18b191d4b4bafa6d4',
    test: 'some_fake_pk...'
}
```
So we can detect network by nethash and write tests for it. Instead of `test` we can use `devnet` or `local`.